### PR TITLE
Add EmberConf reg writeup to issue

### PR DESCRIPTION
**Pull request by amyrlam**
let me know if any feedback!

### CODE REVIEW

File: source/blog/2018-12-21-the-ember-times-issue-78.md [hunk 1]
```diff
@@ -61,8 +60,22 @@ Read and comment on the [full RFC on GitHub](https://github.com/emberjs/rfcs/pul
...
+
+2019 is rapidly approaching, and you've got one more thing to do before EOY: buy your EmberConf ticket! This year's Early-Bird discount is significant ($50), while supplies last or until 12/31. So before anything else, here's that big shiny button! 🆗
+
+<div class="blog-row">
+  <a class="ember-button" style="width:260px" href="https://emberconf.com/register.html">Register Now</a>
+</div>
+
+The program has been announced and is looking really great: so many new faces, and a bunch of familiar old ones! Be sure to check out the [full schedule](https://emberconf.com/schedule.html) for all the details. 
+
+We're particularly excited about [Inclusiveness at EmberConf](https://emberconf.com/inclusiveness-at-emberconf.html) this year. In an industry lacking attendee and speaker diversity, EmberConf strives to be an exception. Highlights of this year's efforts include:
```
Comment (jessica-jordan) [inline source/blog/2018-12-21-the-ember-times-issue-78.md]:
```suggestion
 
 <!--alex ignore gals-men-->
 <!--alex ignore kids-->
```

File: source/blog/2018-12-21-the-ember-times-issue-78.md [hunk 2]
```diff
@@ -61,8 +60,24 @@ Read and comment on the [full RFC on GitHub](https://github.com/emberjs/rfcs/pul
...
+
+<div class="blog-row">
+  <a class="ember-button" style="width:260px" href="https://emberconf.com/register.html">Register Now</a>
+</div>
+
+The program has been announced and is looking really great: so many new faces, and a bunch of familiar old ones! Be sure to check out the [full schedule](https://emberconf.com/schedule.html) for all the details. 
 
+We're particularly excited about [Inclusiveness at EmberConf](https://emberconf.com/inclusiveness-at-emberconf.html) this year. In an industry lacking attendee and speaker diversity, EmberConf strives to be an exception. Highlights of this year's efforts include:
+ 
+ <!--alex ignore gals-men-->
```
Comment (jessica-jordan) [inline source/blog/2018-12-21-the-ember-times-issue-78.md]: Woops, it seems several messages need to be ignored like so:

```suggestion
  <!--alex ignore gals-men kids-->
```

File: source/blog/2018-12-21-the-ember-times-issue-78.md [hunk 3]
```diff
@@ -61,8 +60,24 @@ Read and comment on the [full RFC on GitHub](https://github.com/emberjs/rfcs/pul
...
+ <!--alex ignore kids-->
```
Comment (jessica-jordan) [inline source/blog/2018-12-21-the-ember-times-issue-78.md]:
```suggestion
```

File: source/blog/2018-12-21-the-ember-times-issue-78.md (hunk 2 above)
Comment (amyrlam) [reply source/blog/2018-12-21-the-ember-times-issue-78.md]: fixed it `<!--alex ignore gals-men women kids-->`

**Pull request merged**